srcpkgs/python3-WTForms/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'python3-WTForms'
 pkgname=python3-WTForms
-version=3.0.1
-revision=3
-build_style=python3-module
-hostmakedepends="python3-setuptools python3-wheel python3-Babel"
+version=3.1.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel hatchling python3-Babel"
```

Comment:
It seems `python3-wheel` is not required with `hatchling`.

Reply:
Indeed it is not, it has been removed.